src/Insides/FormatNegotiator.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Franzl\Middleware\Whoops;
+namespace Franzl\Middleware\Whoops\Insides;
```

Comment:
It looks like that `Private` is not a valid part of a namespace. Do you have an alternative proposal?